restful_api/migrations/versions/f221e6b46bdc_.py

```diff
@@ -0,0 +1,33 @@
+"""empty message
```

Comment:
Help me delete this file and regenerate with `flask db migrate -m "create_course_table"`